data/2019_pyladies_praha_spring/tasks/08_slovniky.yaml

```diff
@@ -9,21 +9,36 @@ tasks:
   markdown: |
     Napiš funkci, která pro argumentem zadané číslo `n` vytvoří a vrátí slovník,
     kde jako klíče budou čísla od jedné do `n` a jako hodnoty k nim
-    jejich druhé mocniny.
+    jejich druhé mocniny. Například:
+
+    ```python
```

Comment:
```suggestion
    ```pycon
```

Reply:
Teda, asi minutu jsem přemýšlel, jestli jsem se zbláznil já, nebo ty 😄

Reply:
Já když to poprvé viděl, tak jsem taky myslel, že už té konference šilhám.